src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -421,11 +402,8 @@ public ObjectMetadata storeObject(
         String checksumAlgorithm, long objSize
```

Comment:
close the input stream.

Reply:
Thank you! I have added the try-finally blocks to do so.